Sources/UInt128/UInt128.swift

```diff
@@ -529,8 +529,8 @@ extension UInt128 : UnsignedInteger {}
 // MARK: - Hashable Conformance
 
 extension UInt128 : Hashable {
-    public var hashValue: Int {
-        return self.value.lowerBits.hashValue ^ self.value.upperBits.hashValue
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(self.value.lowerBits.hashValue ^ self.value.upperBits.hashValue)
```

Comment:
This would be better in two separate combine statements instead:
```suggestion
        hasher.combine(self.value.lowerBits.hashValue)
        hasher.combine(self.value.upperBits.hashValue)
```